File: GCGeneral-PHP/GCGeneral/Model/Game.php
<?php 
namespace GCGeneral\Model;

use PhpMyCP\System\Model;
use PhpMyCP\System\System;

class Game extends Model {
	
	public $primaryKey = "game_id";
	public $table = "games";
	public $timestamps = true;
	
	protected $fillable = [
			"title",
			"description",
			"price",
			"genre_id",
			"type",
			"download_url",
			"quantity",
			"deleted"
	];
	
	//Install Schema file for Project
	public static function install() {
		System::installFromSQLFile("game-central.sql");
	}
	
}

File: GCGeneral-Java/src/code/slipswhitley/gcgeneral/model/Game.java
package code.slipswhitley.gcgeneral.model;

import net.minidev.json.JSONObject;

public class Game {

	public final int gameID;
	public final String title;
	public final String description;
	
	public final double price;
	public final String type;
	
	public final int quantity;
	public final String downloadURL;
	
	public Game(JSONObject json) {
		this.gameID = Integer.valueOf(json.get("game_id").toString());
		this.title = (String) json.get("title");
		this.description = (String) json.get("description");
		
		this.price = Double.valueOf(json.get("price").toString());
		this.type = (String) json.get("type");
		
		this.quantity = Integer.valueOf(json.get("quantity").toString());
		this.downloadURL = (String) json.get("download_url");
	}
	
	@Override
	public String toString() {
		return gameID + ": " + title + " - Price: $" + price + "      DownloadType: " + type + "      Quantity Available: " + quantity; 
	}
	

}
